albatross/tune.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2018 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef ALBATROSS_TUNE_H
+#define ALBATROSS_TUNE_H
+
+#include <map>
+#include <vector>
+#include "core/model.h"
+#include "nlopt.hpp"
+
+namespace albatross {
+
+std::vector<ParameterValue> transform_parameters(
+    const std::vector<ParameterValue>& x) {
+  std::vector<ParameterValue> transformed(x.size());
```

Comment:
nit: you could use `std::for_each` here and in the following functions below.

Reply:
done (though I had to use `transform` instead of `for_each` to avoid a copy).